frontend/components/welcomeHeader.jsx

```diff
@@ -17,31 +18,64 @@ const TitleContainer = styled.div`
 `;
 
 const TimerContainer = styled.div`
-  align-self: flex-end;
+`;
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled.div`
+  font-family: 'Madina Script';
+  src: url('MadinaScript.ttf')
+`;
+
+const SolvedContainer = styled.div`
 `;
 
 class WelcomeHeader extends React.Component {
-  render() {
-    const { gameState } = this.props;
+  constructor(props) {
+    super(props);
+    this.numSolved = this.numSolved.bind(this);
+  }
 
-    const { start_time } = gameState || {};
+  numSolved(gameState) {
+    var solvedCounter = 0;
+    if (gameState) {
```

Comment:
this can be simplified to `Object.values(gameState || {})`, since that will return an empty list and reduce to 0